src/ray/gcs/format/gcs.fbs

```diff
@@ -32,8 +32,16 @@ enum SchedulingState:int {
 table TaskTableData {
   scheduling_state: SchedulingState;
   scheduler_id: string;
-  execution_arg_ids: [string];
+  execution_dependencies: string;
```

Comment:
can you explain this change?

Reply:
execution_dependencies is now a serialized protocol buffer

Reply:
protocol buffer or flatbuffer?